clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/security/DcosCredentialMap.groovy

```diff
@@ -0,0 +1,19 @@
+package com.netflix.spinnaker.clouddriver.dcos.security
+
+class DcosCredentialMap {
```

Comment:
Maybe just DcosCredentials? I don't necessarily like calling it a map, since it isn't one.